src/services/collectionService.ts

```diff
@@ -494,32 +497,42 @@ export const removeFromCollection = async (
     files: EnteFile[]
```

Comment:
we can improve this in few ways

- first the caller can send the list of existing collection. So for places where this function called multiple times. we would only get the collection collections once, similarly will calling get uncategorizedCollection we can pass it the list of collection, so that it doesn't have to fetch them from localStorage
- also in the current logic we would call move to collection `files.length` number of times, we can improve this by moving group files to one common collection if its possible